### PR TITLE
(ui):Added Cooking Pot Svg and cooking time

### PR DESCRIPTION
**Pull request by 0xSaksham**
image

Working on the Cooking Time, open to reviews and questions
Completes #767

### CODE REVIEW

**Comment by theborakompanioni** [comment]
Hey @0xSaksham! Is it planned to add animation in this PR?

**Comment by 0xSaksham** [comment]
No, I'll create a different PR for that. 

What I plan is to add a cooking time. Does it sound good?

**Comment by theborakompanioni** [comment]
> No, I'll create a different PR for that.
> 
> What I plan is to add a cooking time. Does it sound good?

Let's leave out the concept of "cooking time" till we have a good understanding if that is even feasible. As stated in the description, waiting time is just the minimum estimation. I'd rather not display it for now. Okay for you, if it is removed (for now)?

**Comment by 0xSaksham** [comment]
> Let's leave out the concept of "cooking time" till we have a good understanding if that is even feasible. 

Sure than I'll remove it then.

**Comment by 0xSaksham** [comment]
@theborakompanioni removed cooking time.

File: src/components/JamPotSvg.module.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+.cooking-time {
+  font-size: 1rem;
+  font-weight: 500;
+  color: #4a4a4a;
+}
+
+.paragraph {
+  margin-bottom: 0;
+  text-align: center;
+}
```
Comment (theborakompanioni) [inline src/components/JamPotSvg.module.css:31]:
```suggestion
```

**Comment by 0xSaksham** [comment]
Done!

**Comment by 0xSaksham** [comment]
image

This looks okay?

**Comment by 0xSaksham** [comment]
@theborakompanioni please take a look. 

Everything we talked about is done now.

**Comment by theborakompanioni** [comment]
> @theborakompanioni please take a look.
> 
> Everything we talked about is done now.

Nice. Only thing left is making it work in light mode :wink:

**Comment by 0xSaksham** [comment]
Okay sir!

**Comment by 0xSaksham** [comment]
#804 PR has all the required changes and DRYer code.

**Pull request closed without merging**